### PR TITLE
Duplicate Current Plate With Objects (Second PR)

### PR DESCRIPTION
**Pull request by eoyilmaz**
As per @SoftFever 's request [here](https://github.com/SoftFever/OrcaSlicer/pull/5516#issuecomment-2338337055) I create a new PR.

# Description

- Addressed #2025 
- Added "Duplicate Current Plate" action to the Edit menu.

# Screenshots/Recordings/Graphs

[Video demo](https://github.com/user-attachments/assets/d6f8f37e-8fad-44b5-b5ab-7c12b75b372b)

## Tests

Run manual tests only...

### CODE REVIEW

File: src/slic3r/GUI/GLCanvas3D.hpp [hunk 1]
```diff
@@ -159,6 +159,7 @@ wxDECLARE_EVENT(EVT_GLCANVAS_PLATE_RIGHT_CLICK, RBtnPlateEvent);
 wxDECLARE_EVENT(EVT_GLCANVAS_REMOVE_OBJECT, SimpleEvent);
 wxDECLARE_EVENT(EVT_GLCANVAS_ARRANGE, SimpleEvent);
 //BBS: add arrange and orient event
+// wxDECLARE_EVENT(EVT_GLCANVAS_DUPLICATE_PLATE, SimpleEvent);
```
Comment (SoftFever) [inline src/slic3r/GUI/GLCanvas3D.hpp:162]: Could we remove these unused commented codes?

**Review by SoftFever — COMMENTED** [review]
Thank you.
Overall looks well.
I made some small suggestions

**Comment by eoyilmaz** [comment]
Thank you, I updated my branch with your review comments.

**Comment by eoyilmaz** [comment]
It should be okay now.

**Review by SoftFever — APPROVED** [review]
LGTM
Thank you very much!

**Pull request merged**